src/GameLogic/Enums/TurnEventType.ts

```diff
@@ -8,6 +8,7 @@ export enum TurnEventType {
 	TurnExecuted = 20, // data: undefined
 	LevelLoaded = 21, // data: undefined
 	TileChanged = 22, // data: [number, number] (x and y coordinate)
+	LevelComplete = 23 // data: undefined
```

Comment:
I think we should put the protagonist who exited in the data for the event so we can create some nice visual effect.